### PR TITLE
default.php for nextcloud themes example

### PR DESCRIPTION
**Pull request by JSoko**
changing it to nextcloud

### CODE REVIEW

File: themes/example/defaults.php [hunk 1]
```diff
@@ -2,7 +2,7 @@
 /**
  * @author Björn Schießle <schiessle@owncloud.com>
  * @author Jan-Christoph Borchardt, http://jancborchardt.net
- * @copyright Copyright (c) 2015, ownCloud, Inc.
+ * @copyright Copyright (c) 2016, ownCloud, Inc.
```
Comment (mar1u5) [inline themes/example/defaults.php]: Should we change the year, and is this the right way to do that :grin: @LukasReschke

Comment (MorrisJobke) [inline themes/example/defaults.php]: It's okay. We also have a script that updates the license headers automatically (based on git history) and updates the year. So this should not be a problem.

**Comment by MorrisJobke** [comment]
Makes sense 👍 

Thanks for your contribution :)

**Comment by mar1u5** [comment]
:+1:

**Comment by mar1u5** [comment]
Backport? Obvious, or not? @karlitschek

**Comment by karlitschek** [comment]
nice. please backport

**Comment by MorrisJobke** [comment]
stable9: https://github.com/nextcloud/server/pull/199

**Pull request merged**